drivers/sensor/tdk/icm45686/icm45686_stream.c

```diff
@@ -152,7 +152,7 @@ static void icm45686_complete_handler(struct rtio *ctx,
 
 	if (should_flush_fifo(read_cfg, int_status)) {
 		uint8_t write_reg = REG_FIFO_CONFIG2_FIFO_FLUSH(true) |
-				    REG_FIFO_CONFIG2_FIFO_WM_GT_THS(true);
+				    REG_FIFO_CONFIG2_FIFO_WM_GT_THS(false);
```

Comment:
Can we make this a dt-property? You'd be able to set it to change the behavior to be, say, fifo-wm-equals, and therefore the driver works for both use-cases.